### DISCUSSION TITLE
blog/how-i-overcome-writer-block-when-preparing-a-research-paper

### DISCUSSION THREAD

**Discussion by giscus** [cesarsotovalero/cesarsotovalero.github.io#43] [Announcements]
# blog/how-i-overcome-writer-block-when-preparing-a-research-paper

Writing a research paper can be a tremendous challenge, even for experienced researchers. I’ve faced the so-called writer’s block many times, especially when starting the writing process. Fortunately, with time I’ve developed personal systems that allow me to deal with this particular issue.

https://www.cesarsotovalero.net/blog/how-i-overcome-writer-block-when-preparing-a-research-paper.html

**Comment by martisak** [comment]
Thank you for these great tips! Agree wholeheartedly with almost everything :).

If there is one thing that don't agree with, it is to write the introduction early. The introduction is probably the most difficult and most important part of a paper, and I find that if I begin with that I start to procrastinate again :). Even better, leave it to a more experienced co-author :D. I start directly with the experiment/study design and then the results sections, and describe the figures and tables you added in Sec. 2. Because these are basically just describing what I've done and what the results are so that's easy. And ticking off a an item from the to-write-list feels so good :). 

Here are a few other tips on "How to Write a Paper in a Weekend" from Prof. Peter Carr that I found useful: https://www.youtube.com/watch?v=UY7sVKJPTMA.

**Reply by cesarsotovalero** [reply]
Hi Martin,
I'm glad that you found this post useful.
I agree that writing the introduction first is not the standard way to go.
However, usually the main message of the paper is not clear at the beginning, and writing a draft of the intro is an exercise to make it more apparent.
Of course the intro will be refined many times, as you well said: “it is the most difficult and most important part of the paper.”
Thanks for the additional tip :)
